### PR TITLE
fix(sql): fixed isnan() function for SSE2/SSE4 platforms

### LINKED ISSUE #1012: Crash: EXCEPTION_ILLEGAL_INSTRUCTION
**Describe the bug**
When I run the following query, QuestDB crashes:

`SELECT sensorID, max(reading) from temperatures;`

Log:

2021-05-13T07:32:47.790498Z I http-server connected [ip=127.0.0.1, fd=1580]
2021-05-13T07:32:53.766413Z I i.q.c.h.p.JsonQueryProcessorState [1532] exec [q='SELECT sensorID, max(reading) from temperatures']
2021-05-13T07:32:53.766472Z I i.q.c.h.p.QueryCache miss [thread=questdb-worker-3, sql=SELECT sensorID, max(reading) from temperatures]
2021-05-13T07:32:53.767357Z I i.q.g.SqlCompiler plan [q=`select-group-by sensorID, max(reading) max from (select [sensorID, reading] from temperatures timestamp (ts))`, fd=1532]
2021-05-13T07:32:53.776687Z I i.q.c.h.p.JsonQueryProcessorState [1532] execute-new [skip: 0, stop: 1000]

 A fatal error has been detected by the Java Runtime Environment:

  EXCEPTION_ILLEGAL_INSTRUCTION (0xc000001d) at pc=0x0000000066a9c192, pid=25432, tid=11796

 JRE version: Java(TM) SE Runtime Environment 18.9 (11.0.8+10) (build 11.0.8+10-LTS)
 Java VM: Java HotSpot(TM) 64-Bit Server VM 18.9 (11.0.8+10-LTS, mixed mode, tiered, compressed oops, parallel gc, windows-amd64)
 Problematic frame:
 C  [libquestdb17581070583141605799.dll+0x1c192]

 No core dump will be written. Minidumps are not enabled by default on client versions of Windows

 An error report file with more information is saved as:

**Environment (please complete the following information):**
 - Windows 10
 - questdb 6.0.0

### PR DESCRIPTION
**Pull request by bluestreak01**
closes #1012

### CODE REVIEW

**Review by eugenels — COMMENTED** [review]
it will not be superfluous to add a comment to CMakeList.txt that we are relying on the `/fp:precise` mode property:  `x != x` evaluates to `true` if x is NaN

**Review by eugenels — APPROVED** [review]

**Review by ideoma — APPROVED** [review]

File: ui/package-lock.json [hunk 1]
```diff
@@ -1,17213 +1,8 @@
 {
   "name": "questdb-console",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (ideoma) [inline ui/package-lock.json:4]: why this change is here?

Comment (bluestreak01) [reply ui/package-lock.json:4]: i have no idea, this file has a life of its own - i ran `mvn package` and here we are

**Review by ideoma — APPROVED** [review]
